examples/stats/web_client_monitoring/app.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * An Express application to receive the web metrics and send to Stackdriver
+ * with the opencensus API.
+ */
+
+"use strict";
+
+const express = require("express");
+const assert = require('assert');
+const process = require("process");
+const bodyParser = require('body-parser');
+// [START web_client_monitoring_imports]
+const { Stats, MeasureUnit, AggregationType } = require('@opencensus/core');
+const { StackdriverStatsExporter } = require('@opencensus/exporter-stackdriver');
+// [END web_client_monitoring_imports]
+
+const app = express();
+app.use(express.static("web_client"));
+app.use(bodyParser.json());
+
+// The project id must be provided for running in a local environment
+const project = process.env.GOOGLE_CLOUD_PROJECT;
+assert(typeof project !== 'undefined' && project,
+  "Please set environment variable GOOGLE_CLOUD_PROJECT to the project id.");
+console.log(`Sending metrics data to project: ${project}`);
+
+// OpenCensus setup
+// [START web_client_monitoring_ocsetup]
+const stats = new Stats();
+const exporter = new StackdriverStatsExporter({projectId: project});
+stats.registerExporter(exporter);
+const mLatencyMs = stats.createMeasureDouble("webmetrics/latency",
+                       MeasureUnit.MS,
+                       "Latency related to page loading");
+const mClickCount = stats.createMeasureInt64("webmetrics/click_count",
+                       MeasureUnit.UNIT,
+                       "Number of clicks");
+const buckets = [0, 1, 2, 3, 4, 5, 6, 8, 10, 13, 16, 20, 25, 30, 40, 50, 65, 80,
+                100, 130, 160, 200, 250, 300, 400, 500, 650, 800, 1000, 2000,
+                5000, 10000, 20000, 50000, 100000];
+const tagPhase = "phase";
+const latencyView = stats.createView(
+  "webmetrics/latency",
+  mLatencyMs,
+  AggregationType.DISTRIBUTION,
+  [tagPhase],
```

Comment:
I think we lost the tag "client" for which the value was "web". Could we please return that and use that tag with every view?

Reply:
Sure. I have added that back with the latest commit